tests/DCPS/UnitTests/XtypesAssignability.cpp

```diff
@@ -1507,6 +1507,942 @@ TEST(MapTypeTest, NotAssignable)
 }
 
+void expect_true_non_alias_to_alias()
+{
+  TypeAssignability test;
+  MinimalAliasType ali_a;
+
+  // Primitive types
+  ali_a.body.common.related_type = TypeIdentifier::make(TK_BOOLEAN);
+  EXPECT_TRUE(test.assignable(TypeObject(MinimalTypeObject(ali_a)), *TypeIdentifier::make(TK_BOOLEAN)));
+  ali_a.body.common.related_type = TypeIdentifier::make(TK_BYTE);
+  EXPECT_TRUE(test.assignable(TypeObject(MinimalTypeObject(ali_a)), *TypeIdentifier::make(TK_BYTE)));
+  ali_a.body.common.related_type = TypeIdentifier::make(TK_INT16);
+  EXPECT_TRUE(test.assignable(TypeObject(MinimalTypeObject(ali_a)), *TypeIdentifier::make(TK_INT16)));
+  ali_a.body.common.related_type = TypeIdentifier::make(TK_INT32);
+  EXPECT_TRUE(test.assignable(TypeObject(MinimalTypeObject(ali_a)), *TypeIdentifier::make(TK_INT32)));
+  ali_a.body.common.related_type = TypeIdentifier::make(TK_INT64);
+  EXPECT_TRUE(test.assignable(TypeObject(MinimalTypeObject(ali_a)), *TypeIdentifier::make(TK_INT64)));
+  ali_a.body.common.related_type = TypeIdentifier::make(TK_UINT16);
+  EXPECT_TRUE(test.assignable(TypeObject(MinimalTypeObject(ali_a)), *TypeIdentifier::make(TK_UINT16)));
+  ali_a.body.common.related_type = TypeIdentifier::make(TK_UINT32);
+  EXPECT_TRUE(test.assignable(TypeObject(MinimalTypeObject(ali_a)), *TypeIdentifier::make(TK_UINT32)));
+  ali_a.body.common.related_type = TypeIdentifier::make(TK_UINT64);
+  EXPECT_TRUE(test.assignable(TypeObject(MinimalTypeObject(ali_a)), *TypeIdentifier::make(TK_UINT64)));
+  ali_a.body.common.related_type = TypeIdentifier::make(TK_FLOAT32);
+  EXPECT_TRUE(test.assignable(TypeObject(MinimalTypeObject(ali_a)), *TypeIdentifier::make(TK_FLOAT32)));
+  ali_a.body.common.related_type = TypeIdentifier::make(TK_FLOAT64);
+  EXPECT_TRUE(test.assignable(TypeObject(MinimalTypeObject(ali_a)), *TypeIdentifier::make(TK_FLOAT64)));
+  ali_a.body.common.related_type = TypeIdentifier::make(TK_FLOAT128);
+  EXPECT_TRUE(test.assignable(TypeObject(MinimalTypeObject(ali_a)), *TypeIdentifier::make(TK_FLOAT128)));
+  ali_a.body.common.related_type = TypeIdentifier::make(TK_INT8);
+  EXPECT_TRUE(test.assignable(TypeObject(MinimalTypeObject(ali_a)), *TypeIdentifier::make(TK_INT8)));
+  ali_a.body.common.related_type = TypeIdentifier::make(TK_UINT8);
+  EXPECT_TRUE(test.assignable(TypeObject(MinimalTypeObject(ali_a)), *TypeIdentifier::make(TK_UINT8)));
+  ali_a.body.common.related_type = TypeIdentifier::make(TK_CHAR8);
+  EXPECT_TRUE(test.assignable(TypeObject(MinimalTypeObject(ali_a)), *TypeIdentifier::make(TK_CHAR8)));
+  ali_a.body.common.related_type = TypeIdentifier::make(TK_CHAR16);
+  EXPECT_TRUE(test.assignable(TypeObject(MinimalTypeObject(ali_a)), *TypeIdentifier::make(TK_CHAR16)));
+
+  // String
+  ali_a.body.common.related_type = TypeIdentifier::makeString(false, StringSTypeDefn(70));
+  EXPECT_TRUE(test.assignable(TypeObject(MinimalTypeObject(ali_a)), *TypeIdentifier::makeString(false, StringLTypeDefn(120))));
```

Comment:
This and some other lines are very long.  Try splitting it across multiple lines.